configs/components/rubygem-strscan.rb

```diff
@@ -0,0 +1,7 @@
+component 'rubygem-rexml' do |pkg, settings, platform|
```

Comment:
The component name seems wrong here, shouldn't it be the name of the file?

Reply:
Thanks Chris, I missed that.